src/config.ts

```diff
@@ -3,13 +3,17 @@ import { ERROR } from "./helper/error";
 const ENV_KEYS = [
   "AUTH_EMAIL",
   "AUTH_PASS",
+  "AUTH_EMAIL_TESTNET",
```

Comment:
We now have to have an account per network in Mercury, so these are the new accounts for tesnet and a different pubnet account for our integrity checks as requested by Mercury.